components/homepage/sections/HomePageCard.tsx

```diff
@@ -0,0 +1,135 @@
+import { Container, Flex, SimpleGrid } from "@chakra-ui/react";
+import { ChakraNextImage } from "components/Image";
+import { LandingDesktopMobileImage } from "components/landing-pages/desktop-mobile-image";
+import { StaticImageData } from "next/image";
+import React from "react";
+import { ChakraNextLink, Heading, Text } from "tw-components";
+
+interface Item {
+  img: StaticImageData;
+  title: string;
+  link: string;
+  description: string;
+}
+
+interface HomePageCardProps {
+  title: string;
+  description: string;
+  introductionTitle: string;
+  image: StaticImageData;
+  mobileImage?: StaticImageData;
+  items: Item[];
+}
+
+interface ItemCardProps {
+  item: Item;
+}
+
+const ItemCard = ({ item }: ItemCardProps) => {
```

Comment:
wait did you code all this again? :D 

Please reuse my `HomeProductCard` component I created the other day